sql/resource_sql_endpoint.go

```diff
@@ -194,6 +194,8 @@ func (a SQLEndpointsAPI) Delete(endpointID string) error {
 func ResourceSqlEndpoint() *schema.Resource {
 	s := common.StructToSchema(SQLEndpoint{}, func(
 		m map[string]*schema.Schema) map[string]*schema.Schema {
+		m["enable_serverless_compute"].Deprecated = "This field is intended as an internal API " +
+			"and may be removed from the Databricks Terraform provider in the future"
```

Comment:
No, it should be done only for Global SQL config. There are a valid reasons for disabling serverless on the specific warehouses.

Reply:
reverted